src/common/const.go

```diff
@@ -63,6 +63,7 @@ const (
 	LDAPGroupBaseDN                   = "ldap_group_base_dn"
 	LDAPGroupSearchFilter             = "ldap_group_search_filter"
 	LDAPGroupAttributeName            = "ldap_group_attribute_name"
+        LDAPGroupMembershipAttribute      = "ldap_group_membership_attribute"
```

Comment:
nit: can you please fix spacing so it aligns with other constants?

Reply:
sure ...